### DISCUSSION TITLE
Error when using hex color with transparency

### DISCUSSION THREAD

**Discussion by levelost** [openlayers/ol-mapbox-style#1215] [General]
Using a hex color with transparency in a style (for example #006179c4). Error:
`Could not parse color from value '#006179c4'`

**Comment by ahocevar** [comment]
Hex colors with opacity are not supported. See the [specification](https://docs.mapbox.com/style-spec/reference/types/#color):

> ```json
> {
>    "line-color": "#ff0",
>    "line-color": "#ffff00",
>    "line-color": "rgb(255, 255, 0)",
>    "line-color": "rgba(255, 255, 0, 1)",
>    "line-color": "hsl(100, 50%, 50%)",
>    "line-color": "hsla(100, 50%, 50%, 1)",
>    "line-color": "yellow"
> }

**Reply by levelost** [reply]
But in fact they work.

**Reply by ahocevar** [reply]
In [CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/hex-color) they do, but as stated above, not in Mapbox styles.

**Comment by levelost** [comment]
Got it. Their editor just translates to RGBA values.
![1](https://github.com/user-attachments/assets/2832290c-bd98-4ae8-919c-2f9bd1bd0297)
